packages/pharos-site/src/components/statics/BestPractices.module.css

```diff
@@ -10,22 +10,38 @@
   display: grid;
   grid-template-columns: max-content 1fr;
   column-gap: var(--pharos-spacing-1-x);
+  margin-bottom: var(--pharos-spacing-one-half-x);
 }
 
 .line_dont {
   border: none;
   height: 2px;
   background-color: var(--pharos-color-feedback-error);
-  margin-bottom: var(--pharos-spacing-three-quarters-x);
+  margin-bottom: var(--pharos-spacing-one-half-x);
 }
 
 .line_do {
   border: none;
   height: 2px;
   background-color: var(--pharos-color-feedback-success);
-  margin-bottom: var(--pharos-spacing-three-quarters-x);
+  margin-bottom: var(--pharos-spacing-one-half-x);
+}
+
+.text_do {
+  color: var(--pharos-color-feedback-success);
+  font-weight: bold;
+}
+
+.text_dont {
+  color: var(--pharos-color-feedback-error);
+  font-weight: bold;
 }
```

Comment:
There appears to be some inconsistency in the naming of these classes—some have a single underscore where others have a double underscore. I thing we've got a mix of trying to use some [BEM](http://getbem.com/introduction/) conventions but also not needing to go too wild with it since we're using CSS-in-JS and modules...either way, internal consistency might be good!